dev-upgrade.sh

```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env bash
+if [ -z "$1" ]
+  then
+    echo "No argument supplied"
```

Comment:
Don"t you want to show an help message? like: `usage: $0 <version>`

Reply:
Of course!